.settings/launch.json

```diff
@@ -0,0 +1,33 @@
+{
+	"version": "0.1.0",
+	// List of configurations. Add new configurations or edit existing ones.  
+	// ONLY "node" and "mono" are supported, change "type" to switch.

```

Comment:
Note that comments are not valid in a JSON file -- will this be parsed correctly?